homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
 SERVICE_SYNC = 'sync'
 
+DEFAULT_NUM_REPEATS = '1'
```

Comment:
We should not add this for now as it can already be achieved differently: by just sending multiple commands.